packages/gatsby-source-contentful/src/gatsby-node.js

```diff
@@ -362,6 +365,18 @@ exports.sourceNodes = async (
     })
   }
 
+  createTypes(
+    schema.buildObjectType({
+      name: `ContentfulTag`,
+      fields: {
+        name: { type: `String!` },
+        contentful_id: { type: `String!` },
+        id: { type: `ID!` },
+      },
+      interfaces: [`Node`],
+    })
+  )
```

Comment:
We should move this out of sourceNodes into schemaCustimization API.

A contentfulTag does not have sys fields? (please add a don't infer to it as well)